### PR TITLE
Slime feed buckling

### PR DESCRIPTION
**Pull request by phil235**
- Slimes now uses buckling to feed and attach themselves on mobs. This removes the silly check in mob/living/Move(), reducing lag.
- Fixes mobs being able to attach slimes onto other mobs.
- Fixes being able to get inside a mech sleeper while buckled (if you buckled during the do_after())

### CODE REVIEW

File: code/modules/mob/living/living.dm [hunk 1]
```diff
@@ -577,9 +577,6 @@ Sorry Giacom. Please don't be mad :(
 		// It's ugly. But everything related to inventory/storage is. -- c0
 		s_active.close(src)
 
-	for(var/mob/living/simple_animal/slime/M in oview(1,src))
-		M.UpdateFeed(src)
-
```
Comment (phil235) [inline code/modules/mob/living/living.dm]: the whole reason of this PR.

**Comment by CosmicScientist** [comment]
:+1:

**Comment by Jalleo** [comment]
:moneybag:

**Comment by Aranclanos** [comment]
Sleepers, cryo tubes, mechas, oddyseus sleeper, admin teleports (get and send to), ethereal jaunt and blink won't break the slime buckling and won't take the slime with you. At the moment you move, the slime will teleport on top of you.
Both mecha climbing and mecha sleeper reject it if there's a slime on top of the mob, but only before the cooldown starts. If the slime attaches after the cooldown started, the bug can happen.

the spell and paper that do the teleportation will take the slime with you, they seem to work fine.
The teleporter leaves the slime behind and the hand teleporter also breaks the buckling, but takes the slime with you because of how the blue portal works.

I didn't test with the away mission teleporter.

**Comment by Razharas** [comment]
@phil235

**Comment by phil235** [comment]
Alright, it should be good. I dealt with all problematic cases mentioned by aran.

**Pull request merged**